### PR TITLE
BUG : fix import function

### PR DESCRIPTION
**Pull request by GaelleLeroux**
Hi @allemangD !

I forgot to manage the pip install in the first pull request so I'm doing it now

Thank you :)

### CODE REVIEW

File: MRI2CBCT_CLI/MRI2CBCT_CLI_utils/resample_create_csv.py [hunk 1]
```diff
@@ -1,6 +1,12 @@
 import SimpleITK as sitk
 import os
-import pandas as pd
+
+try : 
+    import pandas as pd
+except ImportError:
+    from slicer.util import pip_install
+    pip_install("pandas")
```
Comment (allemangD) [inline MRI2CBCT_CLI/MRI2CBCT_CLI_utils/resample_create_csv.py:8]: Why not add this to `install_function()`?

Comment (GaelleLeroux) [reply MRI2CBCT_CLI/MRI2CBCT_CLI_utils/resample_create_csv.py:8]: I don't know, I just forgot. I'm gonna do it thanks !

Comment (GaelleLeroux) [reply MRI2CBCT_CLI/MRI2CBCT_CLI_utils/resample_create_csv.py]: I've made the change !

**Review by allemangD — APPROVED** [review]

**Comment by GaelleLeroux** [comment]
Hi @allemangD  ! 

I saw you accept it but I added a new commit to change a little the UI.
Could you merge it if it's ok ?

Thank you :)

**Comment by allemangD** [comment]
For similar reason to https://github.com/DCBIA-OrthoLab/SlicerAutomatedDentalTools/pull/107#discussion_r1699009967, should not manage `vtk` with `pip_install` or `pip_uninstall`.

I searched for references to vtk and found usage in:

- AREG
- ASO
- FlexReg
- MRI2CBCT

There are also references in these modules, but they are not a problem since they are executed in the Conda environment:

- ALI
- ALI_IOS_utils

I also found another reference to `numpy` in FlexReg; I missed it in my prior search because it was written `'numpy<2'`.

These are removed in c1016dbde8aa4d918546f71df2dd5934c335a1df and 1582bdf27fe7117943113fad3819bdf9a5b362d8

Note I rebased on main after merging the other PRs to be sure to catch all usages, hence the force-push.

**Review by allemangD — APPROVED** [review]

**Pull request merged**